bach/bach/series/series.py

```diff
@@ -858,6 +877,28 @@ def sort_values(self, *, ascending=True):
             return self
         return self.copy_override(sorted_ascending=ascending)
 
+    def sort_by_series(self, by: List['Series'], ascending: Union[bool, List[bool]] = True):
+        """
+        Sort this Series by other Series that have the same base node as the caller.
```

Comment:
```suggestion
        Sort this Series by other Series, that have the same base node as this Series.
```

"the caller" I would understand as the function calling this function.

Reply:
Done here https://github.com/objectiv/objectiv-analytics/pull/995/commits/4fc35dd3db64241f29dc2e7a72eb5183baa5f863